ayon_api/server_api.py

```diff
@@ -335,6 +342,22 @@ def __init__(
         self._site_id = site_id
         self._client_version = client_version
         self._default_settings_variant = default_settings_variant
+
+        if ssl_verify is None:
+            # Custom AYON env variable for CA file
+            ssl_verify = os.environ.get("AYON_CA_FILE")
```

Comment:
Wouldn't `os.environ.get("AYON_CA_FILE", True)` achieve the same as the following lines?

Reply:
True, will change.